roles/profile/defaults/main.yml

```diff
@@ -0,0 +1,30 @@
+#---
+## Version of Profile that is installable by this role
+profile_version: 0.0.0
+
+#
+### Profile installer specific variables.
+profile_version_dir: "{{ profile_version | replace('/', '-') }}"
+profile_branch_dir: "{{ openconext_builds_dir }}/OpenConext-profile-{{ profile_branch | replace('/', '-') }}"
+profile_release_dir: "{{ openconext_releases_dir }}/OpenConext-profile-{{ profile_version_dir }}"
+profile_build_path: "{{ openconext_builds_dir }}/OpenConext-profile-{{ profile_version_dir }}.tar.gz"
+profile_download_url: "https://github.com/OpenConext/OpenConext-profile/releases/download/{{ profile_version }}/OpenConext-profile-{{ profile_version_dir }}.tar.gz"
+profile_current_release_symlink: "{{ openconext_releases_dir }}/OpenConext-profile"
+
+profile_domain: profile.{{ base_domain }}
+
+profile_default_locale: en
```

Comment:
perhaps add some documentation as to what these variables are and why they should be configured and how.